www/src/pages/foundations/elevation.jsx

```diff
@@ -0,0 +1,269 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import SEO from '../../components/SEO';
+import Layout from '../../components/PageLayout';
+import {
+  Button, Form, Container, Input, Toast,
+} from '~paragon-react'; // eslint-disable-line import/no-unresolved
+
+const boxShadowSides = ['down', 'up', 'right', 'left', 'centered'];
+const boxShadowLevels = [1, 2, 3, 4, 5];
+
+const controlsProps = [
+  { key: 'x', name: 'Offset X' },
+  { key: 'y', name: 'Offset Y' },
+  { key: 'blur', name: 'Blur' },
+  { key: 'spread', name: 'Spread' },
+  { key: 'color', name: 'Color' },
+];
+
+const BoxShadowNode = () => {
+  const [showToast, setShowToast] = useState(false);
+
+  const isBoxShadowCopied = (level, side) => {
+    navigator.clipboard.writeText(`@include pgn-box-shadow(${level}, "${side}");`);
+    setShowToast(true);
+  };
+
+  const boxShadowCells = boxShadowLevels.map(level => (
+    boxShadowSides.map(side => (
+      <div
+        key={side}
+        role="button"
+        className={`pgn-doc__box-shadow-cell box-shadow-${side}-${level}`}
+        tabIndex={0}
+        aria-label="Box-shadow cell"
```

Comment:
Does it make sense for each cell to have the same `aria-label`, if they are not exactly the same? Should this string include the side and level so each cell is distinct?

Reply:
Yes, corrected